charts/argo-cd/templates/argocd-repo-server/deployment.yaml

```diff
@@ -44,40 +44,134 @@ spec:
         - entrypoint.sh
         args:
         - argocd-repo-server
-        {{- if or (and .Values.redis.enabled (not $redisHa.enabled)) (and $redisHa.enabled $redisHa.haproxy.enabled) }}
-        - --redis
-        - {{ template "argo-cd.redis.fullname" . }}:{{ .Values.redis.servicePort }}
-        {{- else if and .Values.externalRedis.host .Values.externalRedis.port }}
-        - --redis
-        - {{ .Values.externalRedis.host }}:{{ .Values.externalRedis.port }}
-        {{- end }}
+        {{- with .Values.repoServer.logFormat }}
         - --logformat
-        - {{ default .Values.global.logging.format .Values.repoServer.logFormat }}
+        - {{ . | quote }}
+        {{- end }}
+        {{- with .Values.repoServer.logLevel }}
         - --loglevel
-        - {{ default .Values.global.logging.level .Values.repoServer.logLevel }}
+        - {{ . | quote }}
+        {{- end }}
         {{- with .Values.repoServer.extraArgs }}
-        {{- . | toYaml | nindent 8 }}
+          {{- toYaml . | nindent 8 }}
         {{- end }}
-        {{- if .Values.repoServer.containerSecurityContext }}
-        securityContext: {{- toYaml .Values.repoServer.containerSecurityContext | nindent 10 }}
+        {{- with .Values.repoServer.containerSecurityContext }}
+        securityContext:
+          {{- toYaml . | nindent 10 }}
         {{- end }}
         env:
-        - name: HELM_CACHE_HOME
-          value: /helm-working-dir
-        - name: HELM_CONFIG_HOME
-          value: /helm-working-dir
-        - name: HELM_DATA_HOME
-          value: /helm-working-dir
-          {{- include "argo-cd.redisPasswordEnv" . | nindent 8 }}
           {{- with .Values.repoServer.env }}
-          {{- toYaml . | nindent 8 }}
+            {{- toYaml . | nindent 8 }}
```

Comment:
Please reduce the indent of the env block by 2. We should use the same style inside one yaml file. All lists are defined like this:
```yaml
key:
- listitem1
- listitem2
```
and not like this:
```yaml
key:
  - listitem1
  - listitem2
```

Reply:
Done